lib/tests/testutil.py

```diff
@@ -100,10 +105,23 @@ def setUp(self, override_root=True):
 
         self.app_session = FakeAppSession()
 
+        # Create a MemoryMediaFileStorage instance, and the MediaFileManager
+        # singleton.
+        self.media_file_storage = MemoryMediaFileStorage(MEDIA_ENDPOINT)
+        media_file_manager._media_file_manager = MediaFileManager(
+            self.media_file_storage
+        )
+
+        # Accessing the MediaFileManager requires that _is_running_with_streamlit
+        # is True.
+        streamlit._is_running_with_streamlit = True
```

Comment:
As above, lots of tests rely on the MediaFileManager singleton existing, so we create it in this test subclass.